### DISCUSSION TITLE
How can I bind the ctrl+backspace to backward-kill-word?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [fish-shell/fish-shell#10523] [Q&A]
I tried with the `bind \cb backward-kill-word`, `bind -ckb backward-kill-word`, `bind \ck backward-kill-word`. And they didn't work.

**Comment by faho** [comment]
Please see https://fishshell.com/docs/current/interactive.html#custom-bindings and https://fishshell.com/docs/current/interactive.html#key-sequences.
